curvefs/conf/recycle_whitelist.conf

```diff
@@ -0,0 +1,4 @@
+# recycle_whitelist.conf
+#
+# 白名单文件列表
```

Comment:
```
# recycle_whitelist.conf
#
# 白名单文件列表，每行一个文件路径
# /whitelist/file1.txt
# /whitelist/file2.txt
# /whitelist/dir/
```
it seems that this format is better, why not use it? I think users do not like writing all dirs/files in one line, it is too long to read/write.